src/main/java/com/batix/rundeck/plugins/AnsibleResourceModelSource.java

```diff
@@ -283,7 +283,7 @@ public INodeSet getNodes() throws ResourceModelSourceException {
           }
           node.setTags(tags);
 
-          if (root.has("ansible_lsb")) {
+          if (root.has("ansible_lsb") && root.getAsJsonObject("ansible_lsb").has("description")) {

```

Comment:
Ansible gathers an empty object for `lsb_release` and thus fails with a NullPointerException since the previous worked assumed if the fact is available then a `description` property would be available too. Added a check to see if property would be available as well.

This fixes gathering models using Ansible facts on centos/rhel